package errors

import "errors"

var ErrNotFound = errors.New("recurso n√£o encontrado")
